kytos/utils/napps.py

```diff
@@ -573,4 +573,19 @@ def _ask_openapi():
             if answer.lower() in ['n', 'no']:
                 return False
 
+    def reload(self, all_napps=False, napps=None):
```

Comment:
Hi @llgoncalves If I can suggest, most of the logic of similar code we have simply use a list instead of two variables (a boolean and a list). You could just use the napps argument as a list, and when the caller calls the reload on kytos/cli/commands/napps/api.py with args['all'], you can get entire napps list, and then this reload method on this napps.py only iterates over this list. What do you think?

Reply:
Good idea!